### PR TITLE
remove malfunctions

### PR DESCRIPTION
**Pull request by s-ohnishi**
I forgot to do some of the multiple changes.
I also removed unnecessary comments and aligned quotes.

### CODE REVIEW

**Comment by s-ohnishi** [comment]
> Added line #L52 was not covered by tests

I have multiple alerts like this, what should I do?
> codecov/patch Failing after 1s — 0.00% of diff hit (target 5.06%)

Did I do something wrong?
Should I make a patch and upload it?

**Comment by s-ohnishi** [comment]
I'm preparing the next change (Bootstrap update), but I can't move forward if this is stuck.

**Review by aircodepl — APPROVED** [review]
I would only give these arrays some space for better readability
```array('di'=>null)```
→
```array('di' => null)```

You can install PHP CodeSniffer to your IDE for these.

PM or mention me to approve next PR.

**Review by aircodepl — APPROVED** [review]

**Comment by aircodepl** [comment]
@Jeckerson please approve this PR, thanks.

File: src/Controllers/CompaniesController.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function searchAction(): void
                 $this->request->getPost()
             );
 
-            $this->persistent->searchParams = $query->getParams();
+            $this->persistent->searchParams = array('di'=>null) + $query->getParams();
```
Comment (niden) [inline src/Controllers/CompaniesController.php:52]: Can you use the newer syntax (short) for the array please and also run it through codesniffer (spaces needed before and after `=>`)

File: src/Controllers/ProducttypesController.php [hunk 2]
```diff
@@ -54,7 +54,7 @@ public function searchAction(): void
                 $this->request->getPost()
             );
 
-            $this->persistent->searchParams = $query->getParams();
+            $this->persistent->searchParams = array('di'=>null) + $query->getParams();
```
Comment (niden) [inline src/Controllers/ProducttypesController.php:57]: Same here

**Review by niden — CHANGES_REQUESTED** [review]

**Comment by niden** [comment]
> > Added line #L52 was not covered by tests
> 
> I have multiple alerts like this, what should I do?
> 
> > codecov/patch Failing after 1s — 0.00% of diff hit (target 5.06%)
> 
> Did I do something wrong? Should I make a patch and upload it?

Don't worry about that.

File: src/Controllers/CompaniesController.php (hunk 1 above)
Comment (s-ohnishi) [reply src/Controllers/CompaniesController.php]: I have corrected it as requested.

File: src/Controllers/ProducttypesController.php (hunk 2 above)
Comment (s-ohnishi) [reply src/Controllers/ProducttypesController.php]: I have also modified this as per your request.
ProductsController also had the same part, so I modified that as well.

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @s-ohnishi

**Comment by s-ohnishi** [comment]
I'm the one who should be thanking you.
Now I can move on.

**Pull request merged**